lib/exceptional_synchrony/callback_exceptions.rb

```diff
@@ -38,6 +38,38 @@ def return_exception(*args)
           ex
         end
       end
+
+      private
+
+      def handle_result_error(result)
+        error = result.error
+        if error_is_a_timeout?(error)
+          raise Timeout::Error
+        else
+          raise_failure_for_result(result, error: error)
+        end
+      end
+
+      def raise_failure_for_result(result, error: nil)
+        result_string = truncated_inspect(result)
+        error_string = if error
+                         "ERROR = #{truncated_inspect(error)}; "
+                       end
+        raise Failure,  "#{error_string}RESULT = #{result_string}"
+      end
+
+      def truncated_inspect(obj)
+        inspection = obj.inspect[0..100]
```

Comment:
I was thinking you'd change this to `[0, 100]` and `[0, 92]` below?

Reply:
Changed to [0,101] and [0,92] respectively. Thanks for the catch!